.chloggen/print-config-redaction.yaml

```
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: otelcol
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add unified print-config command with mode support (redacted, unredacted), json support (unstable), and validation support.
+
+# One or more tracking issues or pull requests related to the change
+issues: [11775]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext:
```

```suggestion
subtext: This replaces the `print-initial-config` command. See the `service` package README for more details.
```
I think we should make sure users who might be using the previous command are aware it's replaced. Also would be nice to point users to where to find more.

---

Also noted the alias that remains. I would suggest we remove the alias when the feature-flag is finalized.